### ISSUE TITLE
Strip mining feature for 1.18.2

### ISSUE THREAD

**Issue by beartravels** [cabaletta/baritone#3393]
Hello I noticed there is no way to mine with baritone that's not cheating..... or even hits the grey area between cheating and not, So looking online i found this video that almost does it , https://www.youtube.com/watch?v=WXjP6KEOhS0&t=520s and trying those settings today results in https://www.youtube.com/watch?v=yzxN5osLiW4&t=7s
 now we used to strip mine by going 2 blocks and mining that section out as far as you can reach up, left, and right
![2022-04-15_08 11 40](https://user-images.githubusercontent.com/64866787/163587931-df8ecd6c-8069-4639-ba81-5349987e8b28.png)
and it would be great to expand legit miner to do this, but staying on that center value like tunneling but getting every ore, so we can do something like #stripmine [xyz value or pulls from ~~~] [all (all ores), or by specific Minecraft id]

**Comment by beartravels** [comment]
wow no activity?

**Comment by Connorcrafts21** [comment]
This actually looks like a really cool idea

**Comment by liangforstudy** [comment]
Turn on # legitmine

**Comment by Mathrandom7910** [comment]
> Turn on # legitmine

yes, but he wants to mine in this really weird and inefficient way.

**Comment by liangforstudy** [comment]
Maybe he thinks that this will make the mods this that he's legit mining and not using baritone.

**Comment by beartravels** [comment]
no f the mods its for my own play style

**Comment by beartravels** [comment]
it allows me to easily mine and "not" cheat.

**Comment by beartravels** [comment]
if you watch the video I made youll watch me setting legit mine

**Comment by beartravels** [comment]
1 thing i just thought f becsue im not worred about the mods you can add a script like #tunnel then stop then #mine all then when all mines are gone #tunnel agian

**Comment by thefasteagle** [comment]
# Legitmine still looks super fake. It mines from cave to cave in a way a player could never do (as in going straight to the next cave so any mod would see you as a wallhacker) A real player would strip mine or walk through caves till the end. And them when all the way down find another route or go strip mine. It should be a setting as in it doesn't mine any resources but the one you selected. So basically won't break blocks like cobble or deepslate cobble etc. Only the blocks it's looking for. Until there are no more routes to take.

**Comment by liangforstudy** [comment]
Oh yeah, which reminds me of that time one of the mods that was online that
day on server.elementanimation.com he banned me for using baritone to mine
iron_ore! Wtf? How did they know that I'm using baritone when I have
#legitmine turned on, wtf??

**Comment by thefasteagle** [comment]
It's because "legit mine" mines from cave to cave in a weird wallhacking way. Which looks super sus. Which gets you banned - for sure. I never use it in servers.

**Comment by AutoTTV** [comment]
Is this being added?

**Comment by rycbar0** [comment]
to my knowledge nobody is working on this atm.

**Comment by RebelBrytonThe3** [comment]
This would be very useful. But there is a feature like this but not exactly the same. It does some strange stuff. You can mine down and use freecam with meteor coords to go really far out in one direction. Then do #goto x y x. For example. #goto 385 -56 386.
